pypeit/spectrographs/ldt_deveny.py

```diff
@@ -0,0 +1,399 @@
+        # TODO: Not sure if we will need to adjust these at some point
+        par['calibrations']['wavelengths']['n_first'] = 3  # Default: 2
+        par['calibrations']['wavelengths']['n_final'] = 5  # Default: 4
+        #par['calibrations']['wavelengths']['nlocal_cc'] = 13  # Default: 11
+        #par['calibrations']['wavelengths']['fwhm']= 3.0  # Default: 4.0
+        par['calibrations']['wavelengths']['fwhm_fromlines'] = True
+        par['calibrations']['wavelengths']['rms_threshold'] = 0.5  # Default: 0.15
+        par['calibrations']['wavelengths']['sigdetect'] = 10.  # Default: 5.0
+        # Needed to address ISSUE #1155 when non-echelle spectrographs use
+        #  the wavelength calibration method `reidentify`.
+        par['calibrations']['wavelengths']['ech_fix_format'] = False
+
+        # Slit-edge settings for long-slit data (slit > 90" long)
+        par['calibrations']['slitedges']['bound_detector'] = True
+        par['calibrations']['slitedges']['sync_predict'] = 'nearest'
+        par['calibrations']['slitedges']['minimum_slit_length'] = 90.
+
+        # For the tilts, our lines are not as well-behaved as others', 
+        #   possibly due to the Wynne type E camera.
+        par['calibrations']['tilts']['spat_order'] = 4  # Default: 3
+        par['calibrations']['tilts']['spec_order'] = 5  # Default: 4
+    
+        # Reduction and Extraction Parameters
+        par['reduce']['findobj']['sig_thresh'] = 5.0   # Default: [10.0]
+        
+        # Sensitivity function parameters
+        par['sensfunc']['polyorder'] = 7  # Default: 5
+        
+        return par
+
+    def bpm(self, filename, det, shape=None, msbias=None):
+        """
+        Generate a default bad-pixel mask.
+
+        Even though they are both optional, either the precise shape for
+        the image (``shape``) or an example file that can be read to get
+        the shape (``filename`` using :func:`get_image_shape`) *must* be
+        provided.
+
+        Args:
+            filename (:obj:`str` or None):
+                An example file to use to get the image shape.
+            det (:obj:`int`):
+                1-indexed detector number to use when getting the image
+                shape from the example file.
+            shape (tuple, optional):
+                Processed image shape
+                Required if filename is None
+                Ignored if filename is not None
+            msbias (`numpy.ndarray`_, optional):
+                Master bias frame used to identify bad pixels
+
+        Returns:
+            `numpy.ndarray`_: An integer array with a masked value set
+            to 1 and an unmasked value set to 0.  All values are set to
+            0.
+        """
+
+        # Call the base-class method to generate the empty bpm
+        bpm_img = super().bpm(filename, det, shape=shape, msbias=msbias)
+
+        # msgs.info("Using hard-coded BPM for DeVeny")
```

Comment:
Are there any bad columns on the detector?  If not, this is fine.
If so, I'd suggest masking them..

Reply:
No bad columns.

Reply:
I think you can therefore remove this method and use the default one.